### PR TITLE
Update "PWAs on Oculus 2" blog post

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@tomayac PTAL. Could you please also help to upload the following image files to imgix?

**Comment by tomayac** [comment]
Please find the images here:

```
{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/nmHjO4Tbr01psSEfaaOO.png", alt="ALT_TEXT_HERE", width="800", height="600" %}

{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/jLnuUwLBi73HGXiv3XUy.png", alt="ALT_TEXT_HERE", width="800", height="600" %}
```

File: src/site/content/en/blog/pwas-on-oculus-2/index.md [hunk 1]
```diff
@@ -336,11 +335,12 @@ bubblewrap build
 ```
 
 This will output a file called `app-release-signed.apk`. This file can be installed on the device
-or uploaded to the Meta Quest Store and the Google Play Store.
+or published to the Meta Quest Store, Google Play Store or any other Android app distribution
```
Comment (tomayac) [inline src/site/content/en/blog/pwas-on-oculus-2/index.md:338]:
```suggestion
or published to the Meta Quest Store, Google Play Store or any of the other Android app distribution
```

File: src/site/content/en/blog/pwas-on-oculus-2/index.md [hunk 2]
```diff
@@ -408,23 +423,23 @@ in the companion mobile app. Before you can put your device in developer mode, y
 (or have created) a developer organization in the Oculus Developer Center.
 {% endAside %}
 
-After creating the APK, developers can sideload it to the Meta Quest device using the ADB via USB
-or Wi-Fi:
+After creating the APK file, developers can sideload it to the Meta Quest device using the ADB via
```
Comment (tomayac) [inline src/site/content/en/blog/pwas-on-oculus-2/index.md:426]:
```suggestion
After creating the APK file, you can sideload it to the Meta Quest device using the ADB via
```

**Review by tomayac — APPROVED** [review]
LGTM with tiny nits.

**Comment by FluorescentHallucinogen** [comment]
@tomayac @rachelandrew Done.

**Comment by FluorescentHallucinogen** [comment]
@rachelandrew Friendly ping. 😉

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**